### PR TITLE
Delete merged branches

### PR DESCRIPTION
**Pull request by AdamReifsneider**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # pull-automerge
 
 `lein run [github access token]`
+
+It expects a JSON Constant when run through AWS like this: `{ "userToken": "<USER_TOKEN>" }`
```
Comment (MclaughlinSteve) [inline README.md:5]: I feel like I don't have enough context for this to make sense to me.

File: src/pull_automerge/core.clj [hunk 2]
```diff
@@ -103,13 +110,14 @@
   (println "Merge result:" (result :status))
   (if (result :error)
     (remove-label options org repo pull-number label
-      (str "MERGE ERROR: " (result :error)))))
+      (str "MERGE ERROR: " (result :error)))
+    (delete-branch options org repo branch)))
```
Comment (MclaughlinSteve) [inline src/pull_automerge/core.clj:114]: I may be reading this incorrectly, but if there is an error merging and it removes the automerge tag, will it also delete the branch regardless?

File: src/pull_automerge/core.clj [hunk 3]
```diff
@@ -165,12 +173,12 @@
       (def state (pull :mergeable_state))
       (println "mergeable_state is" state)
 
-      (def merge (fn [] (squash-merge-pull options org repo pull-number (pull :title) label)))
+      (def merge-pr (fn [] (squash-merge-pull options org repo pull-number (pull :title) label ((pull :head) :ref))))
```
Comment (MclaughlinSteve) [inline src/pull_automerge/core.clj:176]: This change might be worth doing in a separate PR just so if something goes wrong with the delete, reverting it doesn't also get rid of the quality of life name change

**Review by MclaughlinSteve — CHANGES_REQUESTED** [review]

File: src/pull_automerge/core.clj (hunk 2 above)
Comment (AdamReifsneider) [reply src/pull_automerge/core.clj:114]: https://clojuredocs.org/clojure.core/if
